test/TestProjects/ResourceRename/Generated/Extensions/ResourceGroupExtensions.cs

```diff
@@ -67,7 +67,7 @@ public static async Task<Response<SshPublicKeyInfo>> GetSshPublicKeyateAsync(thi
         /// <param name="keyData"> SSH public key certificate used to authenticate with the VM through ssh. The key needs to be at least 2048-bit and in ssh-rsa format. &lt;br&gt;&lt;br&gt; For creating ssh keys, see [Create SSH keys on Linux and Mac for Linux VMs in Azure](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-linux-mac-create-ssh-keys?toc=%2fazure%2fvirtual-machines%2flinux%2ftoc.json). </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="sshPublicKeyName"/> is null. </exception>
-        public static Response<SshPublicKeyInfo> GetSshPublicKeyate(this ResourceGroupOperations resourceGroup, string sshPublicKeyName, string path = null, string keyData = null, CancellationToken cancellationToken = default)
+        public static Response<SshPublicKeyInfo> CreateSshPublicKey(this ResourceGroupOperations resourceGroup, string sshPublicKeyName, string path = null, string keyData = null, CancellationToken cancellationToken = default)
```

Comment:
Why are these methods here if?  It looks like SshPublicKey is now a resource?  If so shouldn't it have its own container / operations to host the methods for it?

Reply:
yes, but this test project is not for this - I am unsure why we marked this NonResource in this test project. Do you have context?

Reply:
The test project jsons are only used for validating specific scenarios. The annotation may/not be there for reason. Let's not worry about the directive correctness at this point.